### ISSUE TITLE
Lychee-4.0.2: Wrong version number in 3 places

### ISSUE THREAD

**Issue by LudovicRousseau** [LycheeOrg/Lychee#496]
### Detailed description of the problem
Install Lychee-v4.0.2

I would expect to see `Lychee-front Version:       4.0.2` but I have 4.0.1.

Also the login dialog contains 4.0.0

And the À propos dialog contains 4.0.0

### Steps to reproduce the issue

**Screenshots**
![a propos](https://user-images.githubusercontent.com/398496/80031224-afce4a00-84e9-11ea-9227-ad96fb62630c.png)

![login](https://user-images.githubusercontent.com/398496/80031256-b65cc180-84e9-11ea-9df4-c5702a72a1ce.png)

### Output of the diagnostics
```
    Diagnostics
    -------
    Info: Latest version of PHP is 7.4
    Error: Database is behind file versions. Please apply the migration.
    Warning: Dropbox import not working. dropbox_key is empty.

    System Information
    --------------
    Lychee-front Version:       4.0.1 (c41683e)
    Lychee Version (release):   4.0.2

    DB Version:                 4.0.0
    System:                     Linux
    PHP Version:                7.3
    MySQL Version:              10.3.22-MariaDB-0+deb10u1
    
    Imagick:                    1
    Imagick Active:             1
    Imagick Version:            1690
    GD Version:                 2.2.5
```

### Browser and system
Debian GNU/Linux 10
Firefix 75.0

**Comment by ildyria** [comment]
Please apply the migration.

`php artisan migrate`

This will fix this, this is also mentionned in the Diagnostic:

>     Error: Database is behind file versions. Please apply the migration.

**Comment by LudovicRousseau** [comment]
Ah ok.
But because of https://github.com/LycheeOrg/Lychee/issues/493 this command fails.

Maybe with Lychee v4.0.3?

**Comment by ildyria** [comment]
I actually updated the v4.0.2 files, otherwise the migration would always fail in the release. And that is something I am not comfortable sleeping over. :)
But for sure fixed with the v4.0.3
